src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.Simple.cs

```diff
@@ -8,126 +8,193 @@ namespace System.Formats.Cbor
 {
     public partial class CborReader
     {
+        private const int SizeOfHalf = 2; // the size in bytes of an IEEE 754 Half-Precision float
+
+        /// <summary>
+        ///   Reads the next data item as a single-precision floating point number (major type 7)
+        /// </summary>
+        /// <returns>The decoded <see cref="float"/> value.</returns>
```

Comment:
The style guidelines for docs say to not include the return type in the documentation (yeah, there's a lot of prior art both ways): https://github.com/dotnet/dotnet-api-docs/wiki/Returns:-Methods (Other primitive or string).

All of the simple ones probably just turn into

```suggestion
        /// <returns>The decoded value.</returns>
```

Reply:
While the returns was fixed for ReadSingle(), it looks like there are still a lot of `<see cref=`s floating around in returns elements.  I'd also remove them from summary elements where they're just restating the parameter type.

Reply:
Addressed, thanks.